microshift_getting_started/microshift-architecture.adoc

```diff
@@ -10,22 +11,54 @@ Learn the specifics of {product-title} architecture including design intent, how
 
 [id="microshift-architectural-design_{context}"]
 == Architectural design
-{product-title} is a single-node container orchestration runtime designed to extend the benefits of using containers for running applications to low-resource edge environments. {product-title} contains only the APIs and features essential to operating in edge and small form factor computing environments. The following operational differences from {OCP} can help you understand where {product-title} can be deployed:
+{product-title} is a single-node container orchestration runtime designed to extend the benefits of using containers for running applications to low-resource edge environments. Because {product-title} is primarily as a platform for deploying applications, only the APIs and features essential to operating in edge and small form factor computing environments are included.
+
+For example, {product-title} is tasked with only the following Kubernetes cluster capabilities:
+
+* Networking
+* Ingress
+* Storage
+* Helm
+
+{product-title} also provides the following Kubernetes functions:
+
+* Orchestration
+* Security
+
+{product-title} is optimized when used with a compatible operating system, such as {op-system-ostree-first}. When used together, {product-title} and {op-system-ostree-first} form {op-system-bundle}.
+
+.{product-title} as part of {op-system-bundle}.
+image::microshift-RHDE-Tech-Overview.png[<{product-title} is tasked with only the Kubernetes cluster services networking, ingress, storage, helm, with additional Kubernetes functions of orchestration and security.>]
+
+The following operational differences from {OCP} can help you understand where {product-title} can be deployed:
 
 [id="microshift-differences-ocp_{context}"]
 == Key differences from {OCP}
-* Application-level events and metrics are observation-only
+
 * Devices with {product-title} installed are self-managing
 * Compatible with RPM-OStree-based systems
 * Uses only the APIs needed for essential functions, such as security and runtime controls
 * Enables a subset of commands from the OpenShift CLI (`oc`) tool
+* Does not support workload high availability (HA) or horizontal scalability by the adding of worker nodes
+
+.{product-title} differences from {oke}.
+image::microshift-RHDE-v-OKE.png[<{product-title} is tasked with only the Kubernetes cluster capabilities of networking, ingress, storage, helm, with the additional Kubernetes functions of orchestration and security.>]
+
+Figure 2 shows that {oke} has the same cluster capabilities as {product-title}, and adds the following:
+
+* Install
+* Over-the-air updates
+* Cluster Operators
+* Operator Lifecycle Manager
+* Monitoring
+* Logging
+* Registry
+* Authorization
+* Console
+* Cloud Integration
+* Virtual Machines (VMs)
```

Comment:
Virtual Machines (VMS) are also on the left hand side with MicroShift, but using a different technology. MicroShift uses KVM from RHEL (note that on left hand side lower blue box, there is a "VMs" text). Whereas OpenShift uses "OpenShift Virtualization" to run Virtual Machines. So hey have the same capbility to run VMs, but the management, configuration etc. of these is different (that is also denoted by the VMs ..... line on the left hand side).

I have no good idea how to put this complicated situation into nice text, sorry.

Reply:
Let's try breaking the thought up. Try lines 28 and 59ish in the update.